content-src/components/TopSites/TopSites.js

```diff
@@ -8,8 +8,8 @@ const TopSites = React.createClass({
       <div className="tiles-wrapper">
         {props.sites.map((site) => (<a key={site.url} className="tile" href={site.url}>
           <div className="tile-img-container">
-            {(site.leadImage || site.image) && <div className="tile-img"
-              style={{backgroundImage: `url(${site.leadImage || site.image})`}} />}
+            {(site.leadImage || site.image) &&
+              <div className="tile-img" style={{backgroundImage: `url(${ site.leadImage || site.image })`}} />}
```

Comment:
Same as above, I couldn't figure out exactly how ESLint wanted me to format the `/>` since the `<div>` opened on one line and closed on another, so I put all on one line.